frontend/SearchPane.js

```diff
@@ -36,155 +33,27 @@ class SearchPane extends React.Component {
   _key: (evt: EventLike) => void;
```

Comment:
Should probably move this Flow annotation (and maybe rename this file?)
I didn't really completely clean this code up.